test/README.md

```diff
@@ -0,0 +1,86 @@
+# Rendering Tests
+
+Tests for components that render images. They render images to canvas and compare the result image
+with approved reference images.
+Tests are run in the browser headles mode, uses `mocha-webdriver-runner` to run `mocha` tests
+in browser context.
+
+## What to do first? Where to start?
+
+Before start the work on implementing new features you should run rendering tests first to
+generate actual images and store them as reference images to have possibility of making comparision
+with right images after doing some changes in code.
+
+### Prerequisites
+-   Linux
```

Comment:
Does it really work only on Linux? Is there a reason why the docker image does not work on windows/Mac?

Reply:
There is no fundamental problem with running these on Windows/MacOS but it's not easy.

The man obstacle, is that we rely on `--network host` and it's [not available outside linux](https://docs.docker.com/network/host/).

Mac/Windows workflow would require lot of wiring, in order to open connection from  browser to test server running on localhost.

I think we need followup PR for this.

Some interesting links
 * https://stackoverflow.com/questions/40746453/how-to-connect-to-docker-host-from-container-on-windows-10-docker-for-windows
 * there is `host.docker.internal` -> https://docs.docker.com/docker-for-mac/networking & https://docs.docker.com/docker-for-windows/networking/

So seems, that with new dockers it's not so complex, but i would still propose another PR for this

Reply:
Agreed with your proposal @zbigg , thanks for the explanation.

Reply:
also - thanks Zbyszek answering that question...
I will try to add those changes with new PR.

Reply:
I have tried testing this PR in linux wsl and mac and failed, on both systems with no clear error message. As expected I imagine, but I have not yet verified it with plain linux.

Reply:
@zbigg Could you please verify that again and test on your side?

Reply:
@musculman #1106 will adapt scripts and instructions, so all runs on mac + Docker fo Mac.